acceptance/bundle/python/schemas-support/mutators.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import replace
+
+from databricks.bundles.core import schema_mutator
+from databricks.bundles.catalog import Schema
```

Comment:
Can you link back to the discussion on the volumes PR wrt namespace?

Useful context also here.